components/scream/src/diagnostics/tests/potential_temperature_test.cpp

```diff
@@ -122,23 +119,19 @@ void run(std::mt19937_64& engine)
   ekat::Comm comm(MPI_COMM_WORLD);
 
   // Create a grids manager - single column for these tests
-  auto gm = create_gm(comm,1,num_levs);
+  const int ncols = 1;
+  auto gm = create_gm(comm,ncols,num_levs);
 
   // Kokkos Policy
-  TeamPolicy policy(ekat::ExeSpaceUtils<ExecSpace>::get_default_team_policy(1, 1));
+  TeamPolicy policy(ncols, 1);
```

Comment:
Wait, I said to build the policy manually cause I saw you had `(1,1)`. But now you set the league size to `ncols`. And actually, later on seems like you need also a team size larger than 1. So I would go back to the ekat util, and do
```
auto policy = ekat::ExeSpaceUtils<ExecSpace>::get_default_team_policy(ncols, npacks);
```

Reply:
This is a test, so not too big of a deal. However, a team size of 1 might hide some bugs, so I would recommend a team policy (ncols,npacks).